packages/project-access/src/library/helpers.ts

```diff
@@ -0,0 +1,291 @@
+        if (dependencies?.length) {
+            dependencies.forEach((dependency) => {
+                if (
+                    !reuseLibs.some((lib) => {
+                        return dependency === lib.name;
+                    })
+                ) {
+                    missingDeps.push(dependency);
+                }
+            });
+        }
+    });
+    return missingDeps.join();
+}
+
+/**
+ * Returns the library description.
+ *
+ * @param library - library object
+ * @param projectPath - project path
+ * @returns library description
+ */
+export function getLibraryDesc(library: any, projectPath: string): string {
+    const libraryDesc = library?.library?.documentation;
+    if (typeof libraryDesc === 'string' && libraryDesc.startsWith('{{')) {
+        const desc = libraryDesc.replace(/(^{{)|(}}$)/g, '');
+        return getI18nProperty(
+            join(projectPath, library.library?.appData?.manifest?.i18n?.toString()),
+            desc
+        ).toString();
+    }
+    return libraryDesc.toString();
+}
+
+/**
+ * Returns the library dependencies.
+ *
+ * @param library - library object
+ * @returns array of dependencies
+ */
+export function getLibraryDependencies(library: any): string[] {
+    const result: string[] = [];
+    if (library?.library?.dependencies?.dependency) {
+        let deps = library.library.dependencies.dependency;
+        if (!Array.isArray(deps)) {
+            deps = [deps];
+        }
+        deps.forEach((lib: { libraryName: string }) => {
+            // ignore libs that start with SAPUI5 delivered namespaces
+            if (
+                !ui5Libs.some((substring) => {
+                    return lib.libraryName === substring || lib.libraryName.startsWith(substring + '.');
+                })
+            ) {
+                result.push(lib.libraryName);
+            }
+        });
+    }
+    return result;
+}
+
+/**
+ * Returns the i18n property.
+ *
+ * @param i18nPath - i18n path
+ * @param property - property name
+ * @returns i18n property
+ */
+function getI18nProperty(i18nPath: string, property: string): string {
+    try {
+        const libProperties = propertiesReader(i18nPath);
+        return libProperties.get(property)?.toString() ?? '';
+    } catch (e) {
+        return '';
+    }
+}
+
+/**
+ * Returns the manifest description.
+ *
+ * @param manifest - manifest object
+ * @param projectPath - project path
+ * @returns manifest description
+ */
+export function getManifestDesc(manifest: Manifest, projectPath: string): string {
+    const manifestDesc = manifest['sap.app']?.description;
+    if (typeof manifestDesc === 'string' && manifestDesc.startsWith('{{')) {
+        const desc = manifestDesc.replace(/(^{{)|(}}$)/g, '');
+        const i18n = manifest['sap.app']?.i18n?.toString();
+        if (i18n) {
+            return getI18nProperty(join(projectPath, i18n), desc).toString();
+        }
+    }
+
+    return (manifestDesc ?? '').toString();
```

Comment:
Could we use  `getI18nAppPath` or `getI18nModelPaths` here to simplify the code?

Reply:
I've updated to use `getI18nPropertiesPaths` and `getPropertiesI18nBundle` from `@sap-ux/i18n`